linkml_runtime/utils/schemaview.py

```diff
@@ -919,6 +935,21 @@ def get_elements_applicable_by_prefix(self, prefix: str) -> List[str]:
 
         return applicable_elements
 
+    @lru_cache()
+    def all_aliases(self) -> List[str]:
+        """
+        Get the aliases
```

Comment:
added - I ran into something sort of magical.  I can't specify `alias_predicate` in my test data, instead because the slot `alias_predicate` has a `linkml:alias` slot, `predicate`, I can only use `predicate` in my test data. 
https://github.com/linkml/linkml-model/blob/61a6ad31ea5650b17e0f2d51b8497a3aff34e7cb/linkml_model/model/schema/meta.yaml#L401

Reply:
perhaps we should remove the `alias: predicate` from the slot `alias_predicate` ?